content/posts/2020-05-14/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Serverless Authentication with AWS Amplify: A Practical Guide
```

Comment:
For the test to pass, I think you need to change this to "Serverless Authentication with AWS Amplify - A Practical Guide"